lib/cadet/assessments/assessments.ex

```diff
@@ -762,7 +774,7 @@ defmodule Cadet.Assessments do
         if show_all,
           do: "",
           else:
-            "where s.student_id in (select u.id from users u inner join groups g on u.group_id = g.id where g.leader_id = $1)"
+            "where s.student_id in (select u.id from users u inner join groups g on u.group_id = g.id where g.leader_id = $1) or s.student_id = $1"
```

Comment:
Nice that the backend will now show the grader his own submissions; especially useful in development.